### PR TITLE
Mcafee tie fix

### PR DESCRIPTION
**Pull request by PA-Rweins**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/44989)

## Description
McAfee get reputation might return an error, handled error so a better explanation will be returned.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/McAfee-TIE/Integrations/McAfee-TIE/McAfee-TIE.py [hunk 1]
```diff
@@ -176,6 +176,14 @@ def test():
         client.disconnect()
 
 
+def safe_get_file_reputation(tie_client, hash_param):
```
Comment (yaakovi) [inline Packs/McAfee-TIE/Integrations/McAfee-TIE/McAfee-TIE.py:179]: the proper fix should be to return an `Unknown` score instead of raising an error

**Review by yaakovi — CHANGES_REQUESTED** [review]

File: Packs/McAfee-TIE/Integrations/McAfee-TIE/McAfee-TIE.py (hunk 1 above)
Comment (PA-Rweins) [reply Packs/McAfee-TIE/Integrations/McAfee-TIE/McAfee-TIE.py:179]: Umm but what if McAfee will return some unknown error that we've not encountered yet?
Maybe add demisto log with the error and return unknown score?

Comment (anara123) [reply Packs/McAfee-TIE/Integrations/McAfee-TIE/McAfee-TIE.py:179]: When indicator will not be found, the returned error from the API will be very specific. For example 400 error status or something like this.
You will have to check if it is that error, then return Unknown
Other exception, then fail the command

File: Packs/McAfee-TIE/Integrations/McAfee-TIE/McAfee-TIE.py [hunk 2]
```diff
@@ -176,6 +176,15 @@ def test():
...
+    try:
+        res = tie_client.get_file_reputation(hash_param)
+    except Exception as e:
+        demisto.log("McAfee failed to get file reputation with error: " + str(e))
```
Comment (yaakovi) [inline Packs/McAfee-TIE/Integrations/McAfee-TIE/McAfee-TIE.py:183]:
```suggestion
        demisto.info("McAfee failed to get file reputation with error: " + str(e))
```

File: Packs/McAfee-TIE/ReleaseNotes/1_0_4.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### McAfee Threat Intelligence Exchange
+- Added informative message for ***file*** command error.
```
Comment (yaakovi) [inline Packs/McAfee-TIE/ReleaseNotes/1_0_4.md:4]:
```suggestion
- Fixed an error where the ***file*** command failed for hashes for which there was no analysis.
```

**Review by yaakovi — APPROVED** [review]

**Pull request merged**